doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,226 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-    }
-    // wait for new samples (either sleep or use the waitset)
-}
+# If installed and available in PATH environment variable
+iox-roudi
+# If build from scratch with script in tools
+$ICEORYX_ROOT/build/iox-roudi
 ```
 
-Here we do not check whether we actually have data since we already know there is data available by calling `hasData`.
+Afterwards, we can start the applications which immediately connect to the RouDi via their runtime.
 
-### Untyped API
-
-The untyped API offeres similar capabilities and is hence usable in a similar way. The major difference is that
-neither publisher nor subscriber have any knowledge about the underlying type they send or receive. This means that
-the user is responsible to ensure the data is read correctly, i.e. there is no type safety guaranteed by the API
-itself.
+When the application terminates, the runtime cleans up all resources needed for communication with RouDi. This
+includes all memory chunks used for the data transmission which may still be held by the application.
 
-#### Creating a publisher
 
-When creating an untyped publisher we do not need to specify a data type as template paraemter.
+We now briefly define the main entities of an iceoryx system which were already used in the example above.
 
-```cpp
-iox::popo::UntypedPublisher publisher({"Group", "Instance", "CounterTopic"});
-publisher.offer();
-```
+## RouDi
 
-#### Sending data
+RouDi is an abbreviation for **Rou**ting and **Di**scovery. RouDi takes care of the
+communication setup but does not actually participate in the communication between the publisher and the subscriber.
+RouDi can be thought of as the switchboard operator of iceoryx. One of his other major tasks is the setup of the
+shared memory, which the applications use for exchanging payload data. Sometimes referred to as daemon, RouDi manages
+the shared memory and is responsible for the service discovery, i.e. enabling subscribers to find topics offered by 
+publishers. It also keeps track of all applications which have initialized a runtime and are hence able to use
+publishers or subscribers. To view the available command line options call `$ICEORYX_ROOT/build/iox-roudi --help`.
 
-##### 1. Loan and emplace
+## Runtime
 
-Before sending, we have to loan a chunk of memory to emplace our data.
+Each application that wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with RouDi. Only one runtime object per user process is allowed.
 
+To do so, the following lines of code are required
 ```cpp
-auto result = publisher.loan(sizeof(CounterTopic));
+iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
 ```
+## Creating service descriptions for topics
 
-Since the data type is not known to the publisher, we have to provide the size in bytes of the payload data we
-intend to send.
+A ``ServiceDescription`` in iceoryx represents the data to be transmitted and is uniquely identified by three string
```

Comment:
it represents a topic or channel under which publisher/subscribers can exchange data